### ISSUE TITLE
Question : do you have prow run your pipelines stored in the repository ?

### ISSUE THREAD

**Issue by eddycharly** [tektoncd/pipeline#1494]
I see your pipelines are in the tekton folder, how did you setup prow so that it executes the pipelines defined in the repository ?

I have to configure prowjobs and pipelines manually, and i couldn't find a way to have them defined/versioned in the repository itself.

Would be very helpful if you could share some details about how you did it.
Thanks

**Comment by dlorenc** [comment]
cc @afrittoli @vdemeester 

They just gave a talk on this :)

/kind question

**Comment by eddycharly** [comment]
@dlorenc 👍 just in time
can't wait to read details :)

**Comment by dlorenc** [comment]
If you want to dig through it, the code/scripts should be in http://github.com/tektoncd/plumbing

**Comment by afrittoli** [comment]
@eddycharly the test-infra team has a `tekton-pipeline` agent for prow that basically allows linking `ProwJobs` to Tekton `PipelineRuns`. We tried that out but we decided to not pursue this approach any longer for Tekton's own CI.

Instead we're incrementally working in building a Tekton only based CI for Tekton: it gives us full control and it's a very good dogfooding exercise, that allows us to discover missing features in Tekton and encounter and solve issues. Have a look at https://github.com/tektoncd/plumbing/pull/254 - which implements the first Tekton based CI Job - just a demo one for now.
